commit 6534b704d2a4525e0d46f7c1e99d562b52f6050b (refs/changes/20/2420/1)
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2017-08-17T19:02:09+01:00 (2 years, 2 months ago)
    
    MANATEE-352 Add support for synchronous_commit=off in pg_overrides

diff --git a/package.json b/package.json
index aa1e0b2..67cb9ca 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#45f8cbd2",
+        "manatee": "git+https://github.com/joyent/manatee.git#70930603",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
